<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButtonStart">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>start</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditMs">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>113</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditSteps">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>80</y>
      <width>113</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>20</y>
      <width>68</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ms</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>80</y>
      <width>68</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>steps</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxDir">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>140</y>
      <width>50</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>68</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>dir</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
